code/src/components/AddItemForm.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+export const AddItemForm = () => {
+  const [name, setName] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    dispatch(todos.actions.addItem(name))
+    setName('')
```

Comment:
It would be nice if you add a check here to see if the input field is empty or not, before saving the todo. Right now it's possible to save a blank todo.
`if (!name) return`
for example.